src/tests/backend/specs/enforceLowerCasePadIds.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const assert = require('assert').strict;
+const common = require('../common');
+const padManager = require('../../../node/db/PadManager');
+const settings = require('../../../node/utils/Settings');
+
+describe(__filename, function () {
+  this.timeout(30000);
+  let agent;
+  const cleanUpPads = async () => {
+    const padIds = ['UPPERCASEpad', 'uppercasepad', 'ALREADYexistingPad', 'alreadyexistingpad'];
+    await Promise.all(padIds.map(async (padId) => {
+      if (await padManager.doesPadExist(padId)) {
+        const pad = await padManager.getPad(padId);
+        await pad.remove();
+      }
+    }));
+  };
+
+  before(async function () { agent = await common.init(); });
+  beforeEach(async function () {
+    await cleanUpPads();
+  });
+  afterEach(async function () {
+    await cleanUpPads();
+  });
+
+  describe('not activated', function () {
+    Object.assign(settings, {
+      enforceLowerCasePadIds: false,
+    });
+    it('- do nothing', async function () {
+      const res = await agent.get('/p/UPPERCASEpad');
+      assert.equal(res.status, 200);
+    });
+
+    it('- do nothingg', async function () {
+      await agent.get('/p/UPPERCASEpad')
+          .expect(200);
+    });
```

Comment:
These two are equivalent; delete one of them.

```suggestion
    it('- do nothing', async function () {
      await agent.get('/p/UPPERCASEpad')
          .expect(200);
    });
```

Reply:
applied: e3a9eb75e7d4093ed0f8083201f2068f789f6b16